Actions/CheckForUpdates/CheckForUpdates.ps1

```diff
@@ -90,11 +82,38 @@ $templateInfo = "$templateOwner/$($templateUrl.Split('/')[4])"
 
 $isDirectALGo = IsDirectALGo -templateUrl $templateUrl
 if (-not $isDirectALGo) {
-    $ALGoSettingsFile = Join-Path $templateFolder "*/$repoSettingsFile"
-    if (Test-Path -Path $ALGoSettingsFile -PathType Leaf) {
-        $templateRepoSettings = Get-Content $ALGoSettingsFile -Encoding UTF8 | ConvertFrom-Json | ConvertTo-HashTable -Recurse
+    $templateRepoSettingsFile = Join-Path $templateFolder "*/$RepoSettingsFile"
+    if (Test-Path -Path $templateRepoSettingsFile -PathType Leaf) {
+        $templateRepoSettings = Get-Content $templateRepoSettingsFile -Encoding UTF8 | ConvertFrom-Json | ConvertTo-HashTable -Recurse
         if ($templateRepoSettings.Keys -contains "templateUrl" -and $templateRepoSettings.templateUrl -ne $templateUrl) {
-            throw "The specified template repository is not a template repository, but instead another AL-Go repository. This is not supported."
+            # The template repository is a url to another AL-Go repository (a custom template repository)
```

Comment:
Telemetry would be useful when custom template is being used.

Reply:
agree